.gitignore

```diff
@@ -2,3 +2,5 @@ composer.lock
 vendor
 phpunit.xml
 clover.xml
+.phpunit.result.cache
+.idea/
```

Comment:
Once could argue that this should be in peoples own global git ignore, but I've left this be as it was given by upstream repo and could potentially cause conflicts in the future, if removed and upstream alters further.

But let me know if you'd like it removed. :v:

Reply:
Yeah in general I recommend open source projects avoid these things in gitignore but I would rather make as few changes from upstream as possible :+1: